api/src/main/java/com/exelaration/abstractmemery/domains/Image.java

```diff
@@ -1,28 +1,29 @@
 package com.exelaration.abstractmemery.domains;
 
-public class Image {
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import javax.persistence.Transient;
+import lombok.Data;
 
-    private String fileData;
-    private String fileName;
+@Data
+@Entity
+@Table(name = "images")
+public class Image {
 
-    public Image (String fileData, String fileName) {
-        this.fileData = fileData;
-        this.fileName = fileName;
-    }
+  @Id
+  @GeneratedValue(strategy = GenerationType.IDENTITY)
+  private int id;
 
-    public String getFileData() {
-        return fileData;
-    }
+  @Transient private String fileData;
 
-    public void setFileData(String fileData) {
-        this.fileData = fileData;
-    }
+  private String fileName;
 
-    public String getFileName() {
-        return fileName;
-    }
+  private String fileLocation;
 
-    public void setFileName(String fileName) {
-        this.fileName = fileName;
-    }
-}
\ No newline at end of file
+  // @ManyToOne
```

Comment:
We'll need to add this when we actually create the meme, but not at the moment, so I just commented it out